website/demos/ContextMenu.tsx

```diff
@@ -171,27 +95,65 @@ export default function ContextMenuDemo({ direction }: Props) {
         rowKeyGetter={rowKeyGetter}
         columns={columns}
         rows={rows}
-        renderers={{ rowRenderer }}
         className="fill-grid"
         direction={direction}
+        onCellContextMenu={({ row }, event) => {
+          event.preventDefault();
+          setContextMenuProps({
+            rowIdx: rows.indexOf(row),
+            top: event.clientY,
+            left: event.clientX
+          });
+        }}
       />
-      {createPortal(
-        <div dir={direction}>
-          {/* @ts-expect-error */}
-          <ContextMenu id="grid-context-menu" rtl={direction === 'rtl'}>
-            {/* @ts-expect-error */}
-            <MenuItem onClick={onRowDelete}>Delete Row</MenuItem>
-            {/* @ts-expect-error */}
-            <SubMenu title="Insert Row">
-              {/* @ts-expect-error */}
-              <MenuItem onClick={onRowInsertAbove}>Above</MenuItem>
-              {/* @ts-expect-error */}
-              <MenuItem onClick={onRowInsertBelow}>Below</MenuItem>
-            </SubMenu>
-          </ContextMenu>
-        </div>,
-        document.body
-      )}
+      {contextMenuProps !== null &&
+        createPortal(
+          <menu
+            ref={menuRef}
+            className={contextMenuClassname}
+            style={
+              {
+                top: `${contextMenuProps.top}px`,
+                left: `${contextMenuProps.left}px`
```

Comment:
This should work as well
```suggestion
                top: contextMenuProps.top,
                left: contextMenuProps.left
```